doc/classes/TileData.xml

```diff
@@ -123,6 +137,22 @@
 				Sets the polygons count for TileSet physics layer with index [code]layer_id[/code].
 			</description>
 		</method>
+		<method name="set_constant_angular_velocity">
+			<return type="void" />
+			<argument index="0" name="layer_id" type="int" />
+			<argument index="1" name="velocity" type="float" />
+			<description>
+				Sets the constant angular velocity. This angular velocity is applied to objects colliding with this tile.
+			</description>
+		</method>
+		<method name="set_constant_linear_velocity">
+			<return type="void" />
+			<argument index="0" name="layer_id" type="int" />
+			<argument index="1" name="velocity" type="Vector2" />
+			<description>
+				Sets the constant linear velocity. This linear velocity is applied to objects colliding with this tile. This is useful to create conveyor belts.
```

Comment:
And it could be something similar here:
```suggestion
				Sets the constant linear velocity. This does not move the tile. This linear velocity is applied to objects colliding with this tile. This is useful to create conveyor belts.
```